tests/unit/cli/test_cli_config_file.py

```diff
@@ -159,6 +160,39 @@ def test_callback_with_valid_config_file_path(self):
             self.assertIn(arg, self.saved_callback.call_args[0])
         self.assertNotIn(self.value, self.saved_callback.call_args[0])
 
+    def test_callback_with_config_file_from_pipe(self):
```

Comment:
Looks like `os.mkfifo` is only available only on Unix. Can you update it to skip if it's on Windows?
Example: https://github.com/aws/aws-sam-cli/blob/develop/tests/unit/commands/local/lib/test_stack_provider.py#L336

Reply:
Sorry, I updated my changes just before I saw your comment. I looked a bit around the existing tests and felt that I should mock the call to `os.mkfifo` but your suggestion makes sense as well. What would you prefer?

Reply:
I changed the test and arranged it so that it's skipped when run on Windows.